uzdotie_darbi.html

```diff
@@ -0,0 +1,75 @@
+﻿<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='UTF-8'>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Saimniecīsko darbu uzskaites sistēma</title>
+	<script src="https://code.jquery.com/jquery-3.3.1.js"></script>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.css">
+    <link rel='stylesheet' href='./static/css/main.css'>
+</head>
+<body>
+<div class="container">
+<div class="row">
+<div class="col-md-12">
+<h1> Uzdotie darbi</h1>
+<h3> Anatolijs Liepa</h3>
+<form class="form-inline" action="/action_page.php">
+    <label for="pwd">Parole:</label>
+    <input type="password" class="form-control" id="pwd" placeholder="Enter password" name="pswd">
+    <button type="submit" class="btn btn-primary">Iziet</button>
+  </form>
+</div>
+<br>
+  <table class="table table-bordered">
+  </form>
+  <br>
+  <tr>
+    <th style="width:10%; text-align:center">Pieteikts (datums)</th>

```

Comment:
Stilus nerakstām inline, bet gan .css failos, ko pēc tam importējam galvenē (header).

Reply:
Šis ir vajadzīgs?